templates/ping-service.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app: infinispan-service-ping
     clusterName: {{ include "infinispan-helm-charts.name" . }}
     {{- include "infinispan-helm-charts.labels" . | nindent 4 }}
+    {{- with .Values.deploy.svcLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
We should update `values.yaml` to use an array not an object (I realise now that my original GH issue used object, sorry about that):

```yaml
podLabels: []
resourceLabels: []
svcLabels: [] 
```

The we can just do:
```yaml
{{- range .Values.deploy.podLabels }}
{{ .key }}: {{ .value }}
{{- end }}
```

Similarly, instead of repeating logic across all of the templates. We should create a helper function in `helpers.tpl` to define the pod and service labels, so we can add them to each template as a one-liner.

Something like:

```yaml
{{/*
Pod labels
*/}}
{{- define "infinispan-helm-charts.podLabels" -}}
{{- range .Values.deploy.podLabels }}
{{ .key }}: {{ .value }}
{{- end }}
{{- end }}
```

Reply:
Done!